input/images/mCodeDiagram.svg

```diff
@@ -1,4 +1,4 @@
2Y4e1ASjS78Zh3dj5XJdj8gjfhGN4GJnpfwfDzefWJXGHhXvi43wm3W7GHTZRCGHgRCnB+UyWm/dKmpZaui7yfM10yp+DFID58/vIf0omMo8+mzNpNpsdX/8evqrtodokEKj7n0Yyr6Kbi9DxDT7ny9J/9hboRV/uMvHH5GgakgQFunNG/Dyunv05xn7SapCYYNJ4LLQAVZw1NdmUOjR5HCf/2sfJZ10bEiGK1z3B8jmflpx3xUxOV7kwL58vzOu2mNfDtNBznjU/rPnyEHhGqBwUVJ5NZ5F73wt5AHE4btxaOLmPHu8qbT/SGCBNPsfvCJDDBEidkbzQHUDyubewfjvATgWC0ir53F3WBeXusqKfi2a5pF04enlTMU1JriwH59pjAuesgqYu5FoWm1Mk+XjxAKPh+nh/Wk5gOnVtlZMTDElJyY6rk0nJtjFp2qrDtai05EhV7SzwMf17666Va/U6CNcP3/Q/r59fviuyeOL8EvXbB1Da8QRzu0yaJZjnCe9bU6CR8BGZPmofVbQiH3Nb0Vc3/VxEtCAaStZZV02WY1o6naeJTQrrIPHr3puvgsPSh9dHm3sFaTCMBgYeN/GcoGHmeu0zy1Zm0aKVMF5yzdBPMaeN2V0Nsrv6y9rSgcvoxcPO2xpMacLo/hyG+/NjgAYpPvghStoPB4dwYbIwEdjGF+F5dVFtAqf4cVoAVDmuM9HJLgkayxXJmlo6xmrGsDZPWDuYwxeFBr7efvZny+3q+1HuuOccYm2S1Sjedq4c8HpE4BdfD+hXi1/12r50DWKsQXrdgRifQ6M8nNKx16wivfaVpRhLyhYu84Z0niwMGF1rdTIDntev7Kga1bfW4nUry04b1s3ytomUBWb23jZxTBs+r7Thz0lWRFxnyJuEIrLWG278YQN3STmclqLE6SIitBXO5s2vTl0ZWhKx5vSb+cEmjjHzIyK5YcW7uTM/ShheSrw7T+AGQbj8oW6bUEk0S1SYe/Q6vHavA6T9aIA1BCtIy3vUO6Avl8McLgU+eLIO5RAoT8H6uBHidIglxc8nxku9pqBbhwNLjBtYPA9JjFfMWxAjxs8j66GputFjtgSmHtmpmrrjNPQhOPnyZUPtZoqC4xDmfbupCJy8VJ6ONMw80E7mS6WN/Ko5eCA5oSzXHfPE1EXcvfOOqPS3G04ucTCPJReyTn48wVxna4BYMihdGXV+MmSx8HzDKTu/NVcvr/+5/V378te3w51xufnTDN65OiORVYYMvL291CMbsoX1iWFCEwATabeaftkUMExZ5ol6N4mEFElqhGQEBnEJXi6FuDIhu+rZ4pK2Rwe1rLk+PTiCi8sO9/73Y8Fs81O94X18UqNCQyZklk8qGesMB11niCfG9mSlMgmKHoAyIDu0ExksUtyWGvToEqSw55LPxZzPIZ5tSYarS48tBg2nPTguWaE1INFsaBO1rWgWXVhw3tHerKLyI3OtbbDfeGv6qaHYhNAPT7haetsFWsznl53cJLaFvw02q5ny5O1X3pYp9IsagM+i1SuzaO3KIVo5rwowf9D0Tw8fb35/+/unS/Od9dcLmCt8Y0GEqgCeNUfaeM8TAdVGeoDuMvWAVpMZBOgHGCsx5KmMPt86K2WOwEZBugGT0mj7vNOUuTrtWsVa4HXVlzfWdZYfyoa8F6kvHVjwGqPfbDGCSDDgnXws2iInWfLqBzpQSVXDws2YO9cRJBXxFVjvgKM672sUz6xbndIomNoodMqkwfIuiq9k6k5q5uSnWQmwLrU8a5k4EaVyuCYRFOqmbORsCrw+BwekcaJgkxwGKspGYGmUN8+HeFzMxMe/oYEyaMlovZncilacyFfV+mpzGHqrnHB1nZwrZxhu6pceRrO+s+HkuFTzdu/NDk1qNXti5ke8XlHMPJZBDYeZbamczGfS4bDwmFt8lrnFJd15u7fnHGp+QNM2WrrDyMQX5fTlZATZGUW1HRWDLO2LAAJ+xl68Xtil2KDMc8vTx/18sdCCL8ZfwfUdcGdf1l+lxUc0VcsnB5mky4I7PKKjUGJetqhmR4lGmmoSyzat0tVpKtmYv/YXEqEkKteIze+GZH5XJxrB8o47JAnIk+NbJ0bKinZ2wOZDy991NbTRhpv+5EkbaKnnsjbb2w41ZtlwjIlmqOkPsdyuZSadLDD8LNxe+Wei5aViDAESeaiUNyoDBcJbUhallDMo3S2ldG5VE5Syp06z55hs8LpK0a6CKEjy8biZRk95JkkDuxdlmyyZ1yH0/d2b//739993y9318o8nRf020wJOh9CYLTDwbAE8QrA0XYA1FlxIusAIbYOEtjfrtb9ezS6Q3xslGm9nL+cCcKG32a19xYufQHk8rb8V2vUwhmVEu+7RjpU43R3aUcYQk9QkNOlokxfVi+l0PnlRxbqSBM+gqLwoQLUNs+ERh9Oiqe0ZJAqMrSSXRJTbjrklmjpQNpPPGUJZu082k9QKRyCbAWqGjK7K4zOzVSnuqLsPU3eH+iZ6cW9mEP/OtLTBQ8+gQKVz3qy84eNCuTEt63Y2DT98WCvG3fvfV3HS+ajCn7sKT5Q3WBpDg++svGHEvGFiXuqveDh4h/OBucRJEcaLboVuPdRvj+jWgYNCozRCJsLJ81Ewqa08XjsaT8M2nooB5KyMp/yFbNOWZzlZYj0UzA2xSnns9RRJdtLNrCZn98V04MyYToNMRk291bl4TopDwdVfv25N3udu74d+VHIisXtRsAn209UcHvNDBSqxz0vvCLevSvt98tZH/8K8fL4wr1sqvNrZtciVVRzq6G/sWyBGIXV0QvLhHvIZbdTprYiMx9y21J+2A5o0411tY7x/QlVv8M/XkBXj8jWpAzmOEMmil1THco++FS08s+p2cNZD95WbS/fq9ramAS+m9Vozq900MaQmbwZjbzZ3NoPZNQz8jgx60zLobm0aw6JXWRa9EBDlszak1O/VTZjO02ejhGkp5nmTSiUu64Jdv8drXWA669q8MHV
```

Comment:
the PatientSpecimen diagram link to the PatientSpecimen profile is broken.

Reply:
From where to where?

Reply:
When I click on the PatientSpecimen box in the icon, I get a 404 message. Seems linked to the following URL: http://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/generalize-genomic-specimen/StructureDefinition-mcode-specimen-archetype.html